telescope-wrapper

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
-export QML2_IMPORT_PATH=`pwd`/usr/lib/arm-linux-gnueabihf/qt5/qml/ #phone
-#export QML2_IMPORT_PATH=`pwd`/usr/lib/x86_64-linux-gnu/qt5/qml #desktop
-env
-find .
+
+architecture="$(uname -p)"
+
+if [ $architecture = "x86_64" ] ; then
```

Comment:
I assume you tried this and will merge it anyway but I noticed that on my machine:
"uname -p" returns "unknown"
"uname -m" returns "x86_64"
You can always fix this in another PR.

Reply:
on my desktop:
![image](https://user-images.githubusercontent.com/35935077/62427410-f2998900-b6f2-11e9-8b2b-5850bd80991d.png)
on my phone:
![screenshot20190804_200017546](https://user-images.githubusercontent.com/35935077/62427440-2d9bbc80-b6f3-11e9-813d-a630551be0ab.png)

Both of them are ubuntu machines.